CHANGELOG.md

```diff
@@ -28,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed: Tracking another user's inventory now properly keeps working after a connection loss.
 - Fixed: Sorting the session history and audit log now works properly.
 - Fixed: In Multiworld session, the Claim world button is now properly disabled when you don't have permissions.
+- Removed: Starting sessions is no longer necessary and has been removed as an option.
```

Comment:
this is a very significant UX change and ought to be close to the top of the list imo. possibly even marked as major to ensure everyone sees it